package.json

```diff
@@ -10,10 +10,14 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "localforage": "^1.10.0",
```

Comment:
I see you installed `localforage`, `match-sorter`,  and `sort-by` but you never used them. Is there a reason for that?